test/fixtures/test-runner/snapshots/unit.js

```diff
@@ -22,3 +22,7 @@ test('`${foo}`', async (t) => {
   const options = { serializers: [() => { return '***'; }]};
   t.assert.snapshot('snapshotted string', options);
 });
+
+test('escapes', async (t) => {
```

Comment:
It would be good to add a test where the test name includes the problematic escape characters since that is serialized as the snapshot ID.

Reply:
The test before mine does that, doesn't it? I could combine these, of course, but that one sets a custom serializer, so I didn't want to tamper with it. 🤔

Reply:
Yes, please don't change the existing tests. It would be good to have coverage without custom serializers, and they are pretty cheap to add.